CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Fixed
 
 - Bug were hit count was not correctly calculated for functions declared at same line
+- Bug were functions not from user project were included in coverage report if `SCARB_CACHE` was set
```

Comment:
```suggestion
- Bug where functions not from user project were included in coverage report if `SCARB_CACHE` was set
```
?

I guess we could rephrase it as well, to something like "Functions from imported projects are no longer included in report when `SCARB_CACHE` is set` ? cuz it's a bit like

![obraz](https://github.com/user-attachments/assets/47285061-b0d8-4819-9fac-31f86ec86413)

:D